data/campaigns/Descent_Into_Darkness/scenarios/10_Endless_Night.cfg

```diff
@@ -1906,6 +1906,11 @@ I can still hear his infernal cackling ringing in my head. Sure thing, Zephrin,
     [event]
         name=enemies defeated
 
+        [set_achieved]
+            content_for="Descent into Darkness"
+            id="did_finished"
```

Comment:
DiD's final scenario is weird - this isn't the end of the campaign. The only things that are the end of the campaign are the routes that lead to `next_scenario=null`.

Reply:
You have nevertheless completed all scenarios in the campaign. It would be strange to not have any achievement like that for DiD.

Reply:
I think you won't achieve the wreath in this circumstance, right? While it's not strictly required that the wreath and the "completion" achievement be equivalent, I think it makes sense for them to be.

Reply:
It won't get the wreath.

For the purposes of the achievements PR, I'd suggest using a different campaign. Using TSG would allow giving separate achievements for the two campaign branches.

Reply:
Another option would HttT, or achievements can be offered for the branches, 5A, 5B, 19A, 19B, 19C, 20A, 20B, and even 2 special achievements if Konrad/Lisar take specter of Fire

Reply:
> DiD's final scenario is weird - this isn't the end of the campaign. The only things that are the end of the campaign are the routes that lead to next_scenario=null.

I would like to point that you can complete DiD's Endless Night by 3-ways: (1) Do it 3x times or so, and let some enemy take the book, (2) get demolished by White Fang Orcs, (3) Get demolished by enemy after 3 wins

Reply:
Would it make more sense to just remove the DiD completion achievement, at least for now?

Reply:
Yes, please remove it.

Reply:
It has been removed.